cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -219,45 +205,44 @@ def ssa_compliance_policy(appliance):
 
 
 @pytest.fixture(scope="module")
-def ssa_compliance_profile(appliance, provider, ssa_compliance_policy):
+def ssa_compliance_profile(appliance, ssa_provider, ssa_compliance_policy):
     profile = appliance.collections.policy_profiles.create(
         'ssa_policy_profile_{}'.format(fauxfactory.gen_alpha()), policies=[ssa_compliance_policy])
 
-    provider.assign_policy_profiles(profile.description)
+    ssa_provider.assign_policy_profiles(profile.description)
     yield
-    provider.unassign_policy_profiles(profile.description)
+    ssa_provider.unassign_policy_profiles(profile.description)
     profile.delete()
 
 
 @pytest.fixture(scope="module")
-def ssa_single_vm(request, local_setup_provider, provider, vm_analysis_provisioning_data,
-           appliance, analysis_type):
+def ssa_single_vm(request, ssa_provider, vm_analysis_provisioning_data, appliance,
+                  analysis_type):
     """ Fixture to provision instance on the provider """
     def _ssa_single_vm():
         template_name = vm_analysis_provisioning_data['image']
         vm_name = 'test-ssa-{}-{}'.format(fauxfactory.gen_alphanumeric(), analysis_type)
-        collection = provider.appliance.provider_based_collection(provider)
+        collection = ssa_provider.appliance.provider_based_collection(ssa_provider)
```

Comment:
Required:
you have inherited `appliance` no need of use `provider attribute` here 
`appliance.provider_based_collection(provider)`